squarescale/batch_test.go

```diff
@@ -0,0 +1,235 @@
+package squarescale_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/squarescale/squarescale-cli/squarescale"
+)
+
+func TestGetBatches(t *testing.T) {
+
+	// nominal case
+	t.Run("nominal get batches", nominalCase)
```

Comment:
May rename `nominalCase`, `NotFoundCase` & `InternalServerErrorCase` to avoid conflicts with other tests in package `squarescale_test`